packages/babel-helper-builder-react-jsx-experimental/src/index.js

```diff
@@ -10,6 +10,29 @@ const DEFAULT = {
 };
 
 export function helper(babel, options) {
+  const { useSpread = true, useBuiltIns = false } = options;
+
+  if (typeof useSpread !== "boolean") {
```

Comment:
this is an opposite check to what had to be introduced at an upper level (jsx transform) because from the perspective of this builder the `useSpread: true` is default but we need to keep the compat for public packages which had `useSpread: false` as default